### ISSUE TITLE
Configure Travis Integration

### ISSUE THREAD

**Issue by repjarms** [watson-developer-cloud/assistant-demo#3]
### What area of development does this issue relate to?
- [ ] Design
- [ ] Back end development
- [x] Front end development
- [ ] Q/A

### What is the nature of the issue?
- [ ] Bug report
- [x] Planned task

### What is the timeline for this issue?
- [x] Critical - immediately
- [ ] Current sprint
- [ ] Next Sprint
- [ ] Greater than next sprint

### Please describe the issue

In order to do automatic deployments on merge into master, Travis needs to be reconfigured for the public github. Self assigning.

**Comment by Repjarms** [comment]
@germanattanasio travis is updated and deploying to the test endpoint through the `dev` branch.

One thing that we need before we can merge into `master` is the env variables for the assistant instance that the app will use.

While we were running on the internal github, we encrypted the `.env` file using `travis encrypt-file` and committed the resulting `.env.enc`. Is this what we want to do in the public demo?

Otherwise, we can add the env vars in through the Travis GUI settings and have a step to inject them into a `.env` file while deploying.

**Comment by germanattanasio** [comment]
Please add the environment variables to travis without using the `.env.enc` file.

Make sure integration tests are skipped if there are no credentials otherwise PRs from contributors will fail

**Comment by Repjarms** [comment]
Ok, I will add to the `.travis.yml` to build a `.env` file from settings variables.

For integration tests, why don't we just provision another instance of WA and inject the credentials into a `.env` for tests on Travis? It doesn't seem right to have contributors not subject to the full test suite for the application. IMO, it would be expected of contributors to be pointing to an instance of WA locally while they are developing, and then for them to expect their code to be tested by the same suite that would be run on the automated builds.

**Comment by germanattanasio** [comment]
I like your idea of provisioning a lite plan just for the tests

**Comment by Oliver-I** [comment]
@repjarms @germanattanasio Has this been completed? Can this issue be closed?

**Comment by Oliver-I** [comment]
@repjarms @germanattanasio Has this been completed? Can this issue be closed?

**Comment by germanattanasio** [comment]
@Oliver-I, @stevenpkg what's the name of the app that is running this code? and where is that app running? is it in `WatsonPlatformServices`/`demos`?

**Comment by Oliver-I** [comment]
@germanattanasio The app is located WatsonPlatformServices / demos and is called watson-assistant-demo.

**Comment by Oliver-I** [comment]
@germanattanasio Has this now been resolved?
